sencha-workspace/SlateTasksStudent/app/controller/Dashboard.js

```diff
@@ -136,6 +148,48 @@ Ext.define('SlateTasksStudent.controller.Dashboard', {
         });
     },
 
+    onStudentSelectorSelect: function(combo, rec) {
+        var me = this,
+            studentCombo = me.getStudentSelectorCombo(),
+            params = me.getSearchParams(),
+            username;
+
+        if (!studentCombo) {
```

Comment:
@themightychris This check was done as a precautionary measure. I don't see a reason why the studentCombo wouldn't be available but if it wasn't this would at least handle the error gracefully.

Reply:
@jtwiest In this app there's no normal condition under which it should not exist. For things like this I'd prefer an abnormal condition to generate an error that our top-level error monitoring tools can catch and report than to silently dismiss it. If that component doesn't exist we're off the road and ignoring it isn't handling it. Also, putting guards in for things that shouldn't happen is a bottomless pit. IMHO it's better to just have a good system in place for reporting on unhandled errors. I would remove this guard

Reply:
👍  What top-level error monitoring tools are in place?